crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -2,7 +2,7 @@ mod downsampling_pipeline;
 mod settings;
 mod upsampling_pipeline;
 
-use bevy_color::LinearRgba;
```

Comment:
I wrote this, but I do not like what the new trait requires usage-wise. I want to find a way to not require this trait be included.

This might not be as big an issue if this is already included in prelude, but if people have suggestions to not require this inclusion, that would be appreciated.

Reply:
I think this can absolutely be put in the prelude, it's one of the main use-cases for it actually, importing traits that are mostly used by users.